airflow/providers/amazon/aws/hooks/redshift_sql.py

```diff
@@ -174,3 +175,63 @@ def get_conn(self) -> RedshiftConnection:
         conn_kwargs_dejson = self.conn.extra_dejson
         conn_kwargs: dict = {**conn_params, **conn_kwargs_dejson}
         return redshift_connector.connect(**conn_kwargs)
+
+    def get_openlineage_database_info(self, connection) -> DatabaseInfo:
```

Comment:
What is the type of `connection`?

Reply:
Do you mean it's missing type hint?

Reply:
Missing a type hint, and also (maybe because of that) it was a genuine curiosity because I have no idea what type of connection that is or what it contains.

Reply:
I've added some more type hints. It's Airflow Connection, the one that keeps information how to connect to various services.

Reply:
@o-nikolas  since now there's a requirement to solve all conversations can we solve this one?

Reply:
@JDarDagran -> see https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#step-5-pass-pr-review - you do not have to "wait" to resolve the conversation if you **believe** it is resolved. If you think it is and you addressed it, as author of the PR you can also resolve it yourself.

Reply:
It's still a bit subjective if I think so but probably I'm too meticulous 😄  Thanks for reminding though!